harvard-lts/CURIOSity
# Silence deprecation warnings

**Silence blacklight deprecation warnings**
* * *

# What does this Pull Request do?
Silences blacklight deprecation warnings (NOT rails or other ruby warnings, just blacklight)

# How should this be tested?

I only know how to find these on a stack-run version so the way I tested was deploy this as the trial branch to dev and see that I don't get the warning when following the service logs anymore. I will revert back so you can test:

- use the regular image and browse the site while having this in terminal 
`docker` service logs -f CURIOSITY_curiosity | grep `"DEPRECATION"`
- you will see as you browse a lot of blacklight 8 warnings
- then redeploy with the trial branch (which is this image) and do the same thing and you shouldn't see those again (again there will be the deprecation rails warnings)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
